src/main/java/com/commercetools/sync/commons/models/WaitingToBeResolvedCategories.java

```diff
@@ -0,0 +1,34 @@
+package com.commercetools.sync.commons.models;
+
+import io.sphere.sdk.categories.CategoryDraft;
+import java.util.Set;
+import javax.annotation.Nonnull;
+
+public final class WaitingToBeResolvedCategories<T extends CategoryDraft>
```

Comment:
I think you don't need to be generic in here, why not? 
`public final class WaitingToBeResolvedCategories extends WaitingToBeResolved<CategoryDraft>` ?

Reply:
7a84c6090